crates/matrix-sdk/src/event_handler/mod.rs

```diff
@@ -229,10 +243,11 @@ where
 
 #[doc(hidden)]
 #[derive(Debug)]
-pub struct EventHandlerData<'a> {
+pub struct EventHandlerData<'a, E: ?Sized> {
     client: Client,
     room: Option<room::Room>,
-    raw: &'a RawJsonValue,
+    /// It's complicated.
```

Comment:
It sure is!

Reply:
And still, I should revise this comment before un-drafting the PR 😄

Reply:
Oh, let's keep as is, for the future generation.